05/DB.test.js

```diff
@@ -1 +1,234 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+describe('Database', () => {
+    describe('insert()', () => {
+        it('Should resolve when item added to database',
+            async () => {
+                const db = new DB();
+                const newItem = { a: 1, b: 2 };
+                await expect(db.insert(newItem))
+                    .resolves.toBe(newItem);
+            });
+
+        it('Should return size of DB = 2 when two items inserted',
+            async () => {
+                const db = new DB();
+                await db.insert({ a: 1, b: 2 });
+                await db.insert({ a: 3, b: 4 });
+
+                const dbSize = db._rows.length;
```

Comment:
Jeśli mamy podkreślenie (_) przed nazwą właściwości w obiekcie to przyjmuje się, że jest prywatna tj. nie korzystamy z niej poza konstruktorem/klasą.  Dlatego tutaj należałoby skorzystać z `getRows()`

Reply:
A nawet przeszło mi przez myśl, że to jest pole prywatne (czy niepubliczne?), ale nie wiem dlaczego pomyślałem, że to nie dotyczy testów 😀 . W późniejszych testach wpadłem na to, żeby skorzystać z `getRows()`.
Dzięki!